ros2bag/ros2bag/verb/record.py

```diff
@@ -46,6 +46,9 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
             '-f', '--serialization-format', default='',
             help='rmw serialization format in which the messages are saved, defaults to the'
              ' rmw currently in use')
+        parser.add_argument(
+            '--no-discovery', action='store_true',
```

Comment:
I guess we could leave the argument as is for now given that the description of it mentions that this only applies to during runtime.